src/index.html

```diff
@@ -11,7 +11,129 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" >
+      <fieldset class="section">
+        <legend>Personal informatiom:
+        </legend>
```

Comment:
Check the code style, please
```suggestion
        <legend>Personal informatiom:</legend>
```

or 

```suggestion
        <legend>
          Personal information:
        </legend>
```